### PR TITLE
Inject into blank frames

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:** @sammacbeth 

## Description:

See: https://app.asana.com/0/inbox/1199237043596759/1200327531868645/1200327707684516

## Automated tests:
- [ ] Unit tests
- [x] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

**Review by sammacbeth — APPROVED** [review]
LGTM

File: integration-test/content-scripts/gpc.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/**
```
Comment (kdzwinel) [inline integration-test/content-scripts/gpc.js:1]: Should we add this to privacy test pages? I guess other platforms could also use such a test.

**Review by sammacbeth — COMMENTED** [review]
1. You have a lint error to fix.
 2. It seems there is not a way to add a site exception just for about:blank, it can only be done for all frames. Do you think that gap is an issue, or that when we have an issue with an about:blank frame it's ok to disable the protection for the whole site?

**Comment by jonathanKingston** [comment]
> It seems there is not a way to add a site exception just for about:blank, it can only be done for all frames. Do you think that gap is an issue, or that when we have an issue with an about:blank frame it's ok to disable the protection for the whole site?

I've allowed them to individually be turned off and also a site list too. Both keys don't need to be present in the format also now so we could ditch the update here: https://github.com/duckduckgo/content-blocking-lists/pull/35

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**